public/src/components/campaign/view_campaign.jsx

```diff
@@ -46,55 +62,42 @@ export default class ViewCampaign extends Component {
             {current_campaign ? (<p>{description}</p>) : null}
             <h4>Script Name:</h4>
             {current_script ? (<p>{scriptName}</p>) : null}
-            <Button
-              bsStyle="primary"
-              onClick={() => { history.push({ pathname: `/admin/scripts/${script_id}` }); }}
-            >
-              View Script
-            </Button>
-            <Button
-              bsStyle="primary"
-              onClick={history.goBack}
-            >
-              Cancel
-            </Button>
-            <a className="btn btn-primary" target="_blank" href={`/campaigns/${campaginId}/csv`}>Download CSV</a>
-          </Col>
-        </Row>
-        <Row>
-          <Col md="6">
-            <h3>Call Outcome Distribution</h3>
-            <Doughnut
-              data={outcomeData}
-              options={chartOptions.outcomeDoughnut}
-            />
-          </Col>
-          <Col md="6">
-            <h3>Call Status Distribution</h3>
-            <HorizontalBar
-              data={statusData}
-              options={chartOptions.statusBar}
-            />
+            <ButtonToolbar>
+              <Button
+                bsStyle="primary"
+                onClick={this.handleViewScriptClick}
+              >
+                View Script
+              </Button>
+              <Button
+                bsStyle="primary"
+                onClick={this.handleCancelClick}
+              >
+                Cancel
+              </Button>
+              <a className="btn btn-primary" target="_blank" href={`/campaigns/${campaginId}/csv`}>Download CSV</a>
+            </ButtonToolbar>
           </Col>
         </Row>
-        <h3>Single Choice and Multi-select Responses Distribution</h3>
-        { (responsesData && responsesData.length) ?
-          (
+        {status && status !== 'draft' ? (
```

Comment:
nit: I think we should put this as `(status && status !== 'draft')` just to be more deliberate.

Reply:
Wait you want () added?